src/frontend-admin/src/components/HelloUser/HelloUser.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import styles from './HelloUser.module.css'
+
+interface IHelloUserProps {
+    name: string
+}
+
+export const HelloUser: React.FC<IHelloUserProps> = props => {
+    const { name } = props
+
+    return (
+        <div className={styles.helloMain}>
+            <div className={styles.helloMessage}>Hello, {name}</div>
+
+            {/*This will become an actual dropdown instead of an arrow when the dropdown is done*/}
```

Comment:
Ah, did not realize you intended to use your dropdown component here. Might ignore my comment above then, but the lesson is important: duplicated css is a code smell for A New Component Is Needed